### DISCUSSION TITLE
Support for new Microcontroller in fs-i6x.

### DISCUSSION THREAD

**Discussion by WWFYMN** [OpenI6X/opentx#438] [Q&A] [answered]
Hello, I opened my radio, I purchased it last month and I wanted to install openTX on it, but the microcontroller wasn't an STM32, it was AMP32. It entered DFU mode just fine by bridging the pads and it connected to my laptop.

**Comment by psicho100pro** [comment]
thanks for the info

**Comment by WWFYMN** [comment, marked as answer]
https://github.com/OpenI6X/opentx/issues/365
